docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -1307,4 +1307,4 @@ There are two different models:
 
 {{% /docs/reference %}}
 
-[Data frames]: https://grafana.com/developers/plugin-tools/introduction/data-frames/
+[data frames]: https://grafana.com/developers/plugin-tools/introduction/data-frames/
```

Comment:
I think this happened last time as well, tho I can't remember why. But this should not be changed to lowercase.

Reply:
yes, seems like a bug with the generation of the help file